Package.swift

```diff
@@ -0,0 +1,32 @@
+// swift-tools-version:4.2
+
+import PackageDescription
+
+let package = Package(
+  name: "GEOSwift",
+  products: [
+    .library(name: "GEOSwift", targets: ["GEOSwift"])
+  ],
+  dependencies: [],
+  targets: [
+    .target(
+      name: "GEOSwift",
+      dependencies: ["CLibGeosC"],
+      path: "./GEOSwift/"
+    ),
+    .testTarget(
+      name: "GEOSwiftTests",
+      dependencies: ["GEOSwift"],
+      path: "./GEOSwiftTests/"
+    ),
+    .systemLibrary(
+      name: "CLibGeosC",
+      path: "./CLibGeosC",
+      pkgConfig: "geos_c",
+      providers: [
+        .brew(["geos"]),
+        .apt(["libgeos-3.6.2"])
```

Comment:
I don't think it's a problem now, but we may start getting into weird situations if the cocoapods/mac/linux versions of the lib are using different geos versions. Currently:

cocoapods: 3.7.1
brew: latest (3.7.0)
apt: 3.6.2

I don't see a way around this though other than just being careful to only use features available in all the versions.

Reply:
Yes, it wasn't clear to me that we can do much about this either.  The one good thing is that I think Cocoapods will link against its own version instead of the system library.  So the difference is mostly `brew` versus `apt`.